### ISSUE TITLE
Parse `Por` as Portuguese

### ISSUE THREAD

**Issue by bestnub** [Sonarr/Sonarr#8057]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Current Behavior

<img width="1564" height="168" alt="Image" src="https://github.com/user-attachments/assets/aae3725e-9ffe-414e-bc11-756440fbaf5f" />

It only detects English, Italian and Spanish

### Expected Behavior

Should also detect French, German and Portugies

### Environment

```markdown
- OS: debian
- Sonarr: 4.0.15.2940
- Docker Install: yes
- Using Reverse Proxy: no
- Browser: Firefox
- Database: Sqlite 3.40.1
```

### What branch are you running?

Main

### Trace Logs?

2025-09-08 21:39:07.5|Trace|Http|Req: 2554 [GET] /api/v3/parse?title=The%20Boys%20S02%20Eng%20Fre%20Ger%20Ita%20Por%20Spa%202160p%20WEBMux%20HDR10Plus%20HDR%20HEVC%20DDP%20SGF (from 10.0.0.220 Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:142.0) Gecko/20100101 Firefox/142.0)
2025-09-08 21:39:07.5|Debug|Sonarr.Http.Authentication.ApiKeyAuthenticationHandler|AuthenticationScheme: API was successfully authenticated.
2025-09-08 21:39:07.6|Debug|Parser|Parsing string 'The Boys S02 Eng Fre Ger Ita Por Spa 2160p WEBMux HDR10Plus HDR HEVC DDP SGF'
2025-09-08 21:39:07.6|Trace|Parser|^(?<title>.+?)[-_. ]+?(?:S|Season|Saison|Series|Stagione)[-_. ]?(?<season>\d{1,2}(?![-_. ]?\d+))(?:[-_. ]|$)+(?<extras>EXTRAS|SUBPACK)?(?!\\)
2025-09-08 21:39:07.6|Debug|Parser|Episode Parsed. The Boys - Season 02 
2025-09-08 21:39:07.6|Debug|Parser|Languages parsed: English, Italian, Spanish
2025-09-08 21:39:07.6|Debug|QualityParser|Trying to parse quality for 'The Boys S02 Eng Fre Ger Ita Por Spa 2160p WEBMux HDR10Plus HDR HEVC DDP SGF'
2025-09-08 21:39:07.6|Debug|Parser|Quality parsed: WEBRip-2160p v1
2025-09-08 21:39:07.6|Debug|Parser|Release Group parsed: 
2025-09-08 21:39:07.6|Debug|AggregateLanguages|Selected languages: English, Italian, Spanish
2025-09-08 21:39:07.6|Trace|Http|Res: 2554 [GET] /api/v3/parse?title=The%20Boys%20S02%20Eng%20Fre%20Ger%20Ita%20Por%20Spa%202160p%20WEBMux%20HDR10Plus%20HDR%20HEVC%20DDP%20SGF: 200.OK (17 ms)
2025-09-08 21:39:07.6|Debug|Api|[GET] /api/v3/parse?title=The%20Boys%20S02%20Eng%20Fre%20Ger%20Ita%20Por%20Spa%202160p%20WEBMux%20HDR10Plus%20HDR%20HEVC%20DDP%20SGF: 200.OK (17 ms)
2025-09-08 21:39:10.8|Trace|Scheduler|Pending Tasks: 0
2025-09-08 21:39:17.4|Trace|Http|Req: 2555 [GET] /api/v3/health (from 10.0.0.220 Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:142.0) Gecko/20100101 Firefox/142.0)
2025-09-08 21:39:17.4|Debug|Sonarr.Http.Authentication.ApiKeyAuthenticationHandler|AuthenticationScheme: API was successfully authenticated.
2025-09-08 21:39:17.4|Trace|Http|Res: 2555 [GET] /api/v3/health: 200.OK (0 ms)
2025-09-08 21:39:17.4|Debug|Api|[GET] /api/v3/health: 200.OK (0 ms)

**Comment by markus101** [comment]
> Trace Logs?

> no

Trace logs are not optional. Post the relevant logs of these releases being parsed.

**Comment by thezoggy** [comment]
But are those audio lang or subtitles

**Comment by bestnub** [comment]
Where do I find the trace logs?

**Comment by markus101** [comment]
It's part of the form you filled out when you opened this issue. https://wiki.servarr.com/sonarr/troubleshooting#logging-and-log-files

**Comment by bestnub** [comment]
Added trace logs

**Comment by bestnub** [comment]
@markus101 Why did you rename the Issue to only "Parse `Por` as Portuguese" and not also include German and French?

**Comment by markus101** [comment]
Because those are already  parsing correctly in the v5-develop code base.
